### ISSUE TITLE
Searching Trash

### ISSUE THREAD

**Issue by WarPoet1** [BrandonSnippets/Lunar-Reports#11]
Searching trash/dumpsters, the skill check is so fast it auto fails it before u can even see the wheel for it

**Comment by BrandonSnippets** [comment]
I also got that issue but after search a few the mini game worked fine. can you try some more and let me know.

**Comment by WarPoet1** [comment]
yeah i tried roughly 8-10 dumpsters and trash cans wasn't able to complete them.
